grr/server/grr_response_server/output_plugins/elasticsearch_plugin_test.py

```diff
@@ -153,6 +161,10 @@ def testReadsConfigurationValuesCorrectly(self):
     self.assertFalse(mock_post.call_args[KWARGS]['verify'])
     self.assertEqual(mock_post.call_args[KWARGS]['headers']['Authorization'],
                      'Basic b')
+    self.assertTrue(
+        mock_post.call_args[KWARGS]['headers']['Content-Type'] == 'application/json' or
+        mock_post.call_args[KWARGS]['headers']['Content-Type'] == 'application/x-ndjson'
+    )
```

Comment:
If this fails in a test, the developer is not provided with a helpful error message, because the test framework only receives the `True` / `False` value . Use `assertIn`:

```
self.assertIn(mock_post.call_args[KWARGS]['headers']['Content-Type'], ('application/json', 'application/x-ndjson'))
```